packages/async-rewriter/src/symbol-table.ts

```diff
@@ -20,22 +20,25 @@ export default class SymbolTable {
       this.add(s, { type: 'classdef', returnType: this.types[s], lib: true });
     });
   }
-  deepCopy(): SymbolTable {
```

Comment:
Can ignore these changes, that just allows us to instantiate the Symbol Table and add global objects in two separate steps instead of one.